bookdown/01-mlr3-basics.Rmd

```diff
@@ -1,28 +1,31 @@
 # mlr3 Basics {#basics}
 
-This chapter will teach you the essential building blocks, R6 classes and operations of `r mlr_pkg("mlr3")`.
-These include creating supervised ML tasks like classification/regression, performing training/prediction and cross-validate/benchmark different models.
+This chapter will teach you the essential building blocks, R6 classes, and operations of `r mlr_pkg("mlr3")`.
+This includes creating supervised machine learning tasks like classification and regression, training models and getting prediction on new data, and evaluating and comparison of different models through cross-validation and benchmarking.
 
-A typical (simple) machine-learning workflow looks like this:
+A typical machine learning workflow looks like this:
 
 ```{r 01-mlr3-basics-001, echo = FALSE}
 knitr::include_graphics("images/ml_abstraction.png")
 ```
 
-Data is usually split up into training and testing portions.
-Using a learning algorithm we induce a model on training data, label the test instances with our model, and compare true test label to predicted labels.
-We obtain a scalar numeric score which quantifies the predictive power of our learner given the current data situation.
+The data, which `r mlr_pkg("mlr3")` encapsulates in tasks, is split into non-overlapping train and test sets to be able to evaluate models objectively &mdash; we are interested in models that generalize to new data rather than just memorizing the training data.
+The training data is given to a machine learning algorithm, called a learner in `r mlr_pkg("mlr3")`, that uses it to build a model of how the features of the data relate to the target values.
+This model is then used to produce predictions on the test data, which are compared to the ground truth values to assess the quality of the model.
+`r mlr_pkg("mlr3")` offers a number of different measures to quantify this quality; usually a numeric score.
```

Comment:
> usually a numeric score

If we mention it explicitly, should we also give an example? Which measures are not a numeric score?

Reply:
In particular I was thinking of confusion matrices. Yes, they're not really measures as such, but they fall into the same category. That said, I don't feel strongly about this if you want to change it.

Reply:
i would keep the matrices and measures separate. both should be covered. but in terms of mlr3 a measure always returns a scalar 1-number score